test/Utilities/functions.jl

```diff
@@ -397,7 +397,10 @@ const MOIU = MOI.Utilities
                     MOI.SingleVariable(x)
                 end
                 @test f ≈ MOI.ScalarQuadraticFunction(MOI.ScalarAffineTerm.([3], [x]),
-                                                      MOI.ScalarQuadraticTerm.([1, 2, 3], [x, y, x], [x, y, y]), 10) - 3
+                                              MOI.ScalarQuadraticTerm.([1, 2, 3], [x, y, x], [x, y, y]), 10) - 3
+                @test f ≈
```

Comment:
Why not simply `@test f ≈ 2.0 * f / 2.0` ?

Reply:
I honestly am not familiar with Julia enough, I was scared that might be optimised out by the compiler at run time. I also looked at the other tests, saw you didn't do `@test f ≈  f + 3 - 3`, and figured that must've been for a reason.

Reply:
Assuming that multiplication works, `f ≈ 2.0 * f / 2.0` correctly tests division. For the starting tests, you cannot assume that any operation works so you need to start with the more verbose style, e.g. `f + 3 ≈ MOI.ScalarAffineFunction(...)`. Then `f ≈ (f + 3) - 3` should be a decent test for `-`

Reply:
That makes sense, I'll change it.

Reply:
Oh, I just realised why I didn't do that. `f` is defined as a `ScalarQuadraticFunction{Int64}`, while `2.0` is a `Float64`.

Reply:
Indeed, it works because `isapprox` works for functions of different coefficient type `T`